chore(rtl): clearly state that rtl and ltr cannot be mixed (#2203)
telerik/blazor-docs: globalization/rtl-support.md

## Configuration

> Enabling the right-to-left support is configured on the root level so it affects all UI for Blazor components in the application. You cannot mix right-to-left and left-to-right orientation in the same application.

Reviewer: ```suggestion
Right-to-left support is configured at the root level so it affects all UI for Blazor components in the application. You cannot mix right-to-left and left-to-right orientation in the same application.
```

> To enable the Right-to-Left direction of the components in your application, set the `EnableRtl` parameter of the [`TelerikRootComponent`]({%slug rootcomponent-overview%})  to `true`.

Reviewer: ```suggestion
To enable right-to-left direction of the components in your application, set the `EnableRtl` parameter of the [`TelerikRootComponent`]({%slug rootcomponent-overview%})  to `true`.
```
